code/modules/reagents/Chemistry-Reagents/New-Reagents.dm

```diff
@@ -97,6 +97,76 @@ datum/reagent/nicotine/on_mob_addicted(var/mob/living/M as mob)
 	data++
 	..()
 	return
+datum/reagent/meth
+	name = "Methamphetamine"
+	id = "meth"
+	description = "Baby Blue."
+	color = "#3333FF" // rgb: 205,133,63
+	addiction_rate = 0.2 
```

Comment:
this is the most ugly addiction system I've ever seen

move it to the goonchem addiction framework, it'll work better + be nicer to read